src/components/timeline/Selection.js

```diff
@@ -356,6 +356,12 @@ class TimelineRulerAndSelection extends React.PureComponent<Props> {
         (committedRange.end - committedRange.start)) *
       width;
 
+    if (!Number.isFinite(selectionStart) || !Number.isFinite(selectionEnd)) {
```

Comment:
I think I'd put that before the 2 const above, as the first thing happening in the function. Maybe add a comment that this prevents a crash too?

Reply:
Yeah, that's a good idea, changed it. Thanks!